charts/newrelic-infra-operator/values.yaml

```diff
@@ -24,7 +24,7 @@ image:
 admissionWebhooksPatchJob:
   image:
     repository: jettech/kube-webhook-certgen
-    tag: v1.5.0
```

Comment:
I'd rather switch to `k8s.gcr.io/ingress-nginx/kube-webhook-certgen:v1.0`, as we discussed internally.

Reply:
Do we have a source for release notes (or versions) for that image? I'm not able to find it since it's quite new :(

Reply:
I can't find one at the moment :/